Sources/HipMobileUI.Droid/Contracts/DroidAudioPlayer.cs

```diff
@@ -88,8 +88,7 @@ private void ShowNotification(bool setPlayImage)
             var mainActivity = (MainActivity) CrossCurrentActivity.Current.Activity;
 
             var builder = new NotificationCompat.Builder(mainActivity)
-                .SetOngoing(true)
-                .SetSmallIcon(Resource.Drawable.ic_headset_white);
```

Comment:
This change looks wrong. `NotificationCompat` is a class related to displaying system notifications, so nothing in the app UI itself. Also, Android notifications *must* have an icon if I remember correctly.